powerline/selectors/tmux.py

```diff
@@ -0,0 +1,34 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+from powerline.bindings.tmux import get_tmux_output
+
+
+_cached_clients = {}
+
+def tmux_via_ssh(pl, segment_info, mode):
+	'''Returns True if tmux client is connected via SSH
+	
+	Requires the ``psutil`` module.
+	
+	Works only for tmux versions >= 2.1'''
+	try:
+		import psutil
+	except ImportError:
+		pl.warn('Module "psutil" is not installed, thus we cannot detect whether tmux is run via ssh')
```

Comment:
This check should be performed at module start. See `cpu_load_percent` segment in powerline/segments/common/sys: depending on whether import is successfull it either creates an actual class or a dummy which does nothing, but a warning with `None` return.